extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php

```diff
@@ -94,4 +94,24 @@ protected function createCNWUrlFromGlobalTitle() {
 			WikiService::WIKIAGLOBAL_CITY_ID
 		)->getFullURL();
 	}
+
+	public function getMenuNodes2016() {
+		$hubsNodes = ( new NavigationModel( true /* useSharedMemcKey */ ) )->getTree(
+			NavigationModel::TYPE_MESSAGE,
+			'global-navigation-menu-hubs',
+			[3] // max 3 links
+		);
+		$exploreNodes = ( new NavigationModel( true /* useSharedMemcKey */ ) )->getTree(
+			NavigationModel::TYPE_MESSAGE,
+			'global-navigation-menu-explore',
+			[1, 3] // max 1 entry with 3 links
+		);
+
+		$menuNodes = [
```

Comment:
we don't but for me this is more readable this way. Instead of having huge array definition which IMHO is hard to follow when it expands to 8 lines - we have two separated functions with readable output.